x-pack/test/api_integration/apis/security_solution/timeline.ts

```diff
@@ -6,9 +6,11 @@
 
 import expect from '@kbn/expect';
 
+// @ts-expect-error
 import { timelineQuery } from '../../../../plugins/security_solution/public/timelines/containers/index.gql_query';
 import {
   Direction,
```

Comment:
Shall we use the `Direction` from x-pack/plugins/security_solution/common/search_strategy/security_solution

Reply:
yes, are you ok to include that in your PR?